private-templates-service/client/src/store/page/reducer.ts

```diff
@@ -13,12 +13,14 @@ const initalState: PageState = {
 }
 
 export function pageReducer(state = initalState, action: PageAction): PageState {
+  // console.log(state);
   switch (action.type) {
     case NAVIGATION:
       return {
         ...state,
         currentPageTitle: action.currentPageTitle,
-        currentPage: action.currentPage
+        currentPage: action.currentPage,
+        previousPage: state.currentPage
```

Comment:
why do you need previous page?

Reply:
there should be a way to access previous page with react router

Reply:
there's no way to access previous page either with history or router